bokehjs/src/coffee/tool/inspectors/inspect_tool.coffee

```diff
@@ -28,7 +28,8 @@ class InspectTool extends Tool.Model
   event_type: "move"
 
   nonserializable_attribute_names: () ->
-    super().concat(['event_type', 'inner_only'])
+    attrs = _.without(super(), 'active')
+    return attrs.concat(['event_type', 'inner_only'])
```

Comment:
Well, I was referring to the python side. Specifically, only one tool from each gesture tool can be active at a time, so it doesn't seem like a single default value will do. I thought you were proposing `Override` to selectively turn on only some of the tools by default. There are combinations that don't make sense, though, turning on pan and box zoom e.g. is impossibly at the same time, since they are both drag tools. What is the right thing to do in that case?